keyboards/sofle_pico/info.json

```diff
@@ -0,0 +1,244 @@
+{
+    "manufacturer": "Ryan Neff",
+    "keyboard_name": "Sofle Pico",
+    "maintainer": "Ryan Neff <github@JellyTitan.com>",
+    "bootloader": "rp2040",
+    "bootloader_instructions": "Hold the BOOT button while plugging in the usb cable",
+    "build": {
+        "lto": false
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP14", "pin_b": "GP15", "resolution": 4}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "dynamic_tapping_term": true,
+        "encoder": true,
+        "encoder_map": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true,
```

Comment:
If OLEDs are enabled at the keyboard level, there must be some code at the keyboard level to support them in a `sofle_pico.c` file. It would be preferable to move the OLED code from the default keymap to the keyboard level.

Reply:
Thanks! I moved the OLED code from the default keymap to the keyboard level, using the `sofle_choc` as an example.